sdk/keyvault/keyvault-admin/test/internal/serviceVersionParameter.spec.ts

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as assert from "assert";
+import { createSandbox, SinonSandbox, SinonSpy } from "sinon";
+import { KeyVaultAccessControlClient, KeyVaultBackupClient } from "../../src";
+import { LATEST_API_VERSION } from "../../src/constants";
+import { HttpClient, WebResourceLike, HttpOperationResponse, HttpHeaders } from "@azure/core-http";
+import { ClientSecretCredential } from "@azure/identity";
+import { env } from "@azure/test-utils-recorder";
+
+// Adding this to the source would change the public API.
+type ApIVersions = "7.2-preview";
+
+describe("The keyvault-admin clients should set the serviceVersion", () => {
+  const keyVaultUrl = `https://eastus2.keyvault_name.managedhsm-int.azure-int.net`;
```

Comment:
Just making sure.. this is meant to be public?

Reply:
We didn't do it in .Net, but I can do it! I'll do it!

Reply:
No, if it is hardcoded in the code, it is fine.